lcs/Perception.py

```diff
@@ -27,3 +27,12 @@ def __len__(self) -> int:
 
     def __repr__(self):
         return ' '.join(map(str, self))
+
+    def __setitem__(self, i, o):
+        self._items[i] = o
+
+    def __eq__(self, other):
```

Comment:
Czy to nam potrzebne? Nie zadziała z automatu?

Reply:
Właśnie też się zdziwiłam, ale niestety nie działa z automatu.